yesod-test/Yesod/Test.hs

```diff
@@ -395,6 +395,13 @@ bodyContains text = withResponse $ \ res ->
   liftIO $ HUnit.assertBool ("Expected body to contain " ++ text) $
     (simpleBody res) `contains` text
 
+-- | Assert the last response doesn't have the given text. The check is performed using the response
+-- body in full text form.
```

Comment:
Can you add a `@since` comment, as well as bump the .cabal version field and add a comment to the ChangeLog.md? This looks good.